lib/vanilladatabase.cpp

```diff
@@ -72,170 +72,200 @@ qint8 getVanillaOrder(int mapId) {
+    { false , false , "Special bonus! You receive 77 times the number of shops you own in gold coins from the bank!"                     },
+    { false , false , "Cashback venture! You can sell a shop back to the bank for 500G more than its shop value."                        },
+    { false , false , "Special bonus! You receive 100 times the number of shops you own in gold coins!"                                  },
+    { true  , false , "Moneymaking venture! Roll the die and get the number shown x your level x 20G from the bank!"                     },
+    { true  , false , "Moneymaking venture! Take your level times 40G from all other players!"                                           },
+    { true  , false , "Misadventure! All other players can only move forward 7 on their next turn."                                      },
+    { true  , false , "Moneymaking venture! Roll the die and get 60 times the number shown in gold coins from the player in 1st place!"  },
+    { true  , false , "Adventurous turning point! Everyone gets to choose which way to move on their next go."                           },
+    { false , false , "Lucky venture! You get a really big commission until your next turn!"                                             },
+    { false , false , "Misadventure! You give 20% of your ready cash to the player in last place!"                                       },
+    { false , false , "Misadventure! You drop your wallet and lose 20% of your ready cash!"                                              },
+    { false , false , "Capital venture! You can invest up to 400G of the bank's money in your shops."                                    },
+    { false , false , "Moneymaking venture! The winning player must pay you 20% of their ready cash!"                                    },
+    { false , false , "Dicey adventure! Roll 1/3/5 and your shops expand by 20%. Roll 2/4/6 and everyone else's shops expand by 5%."     },
+    { false , false , "Suit venture! Buy a Suit Yourself card for 50G."                                                                  },
+    { false , false , "Dicey adventure! Roll 1/3/5 to warp to a boon square. Roll 2/4/6 to warp to the arcade."                          },
+    { false , false , "Revaluation venture! Roll the die and expand your shops by 2% for each number."                                   },
+    { false , false , "Special arcade adventure! You're invited to play Round the Blocks and Memory Block!"                              },
+    { false , false , "Special bonus! You receive 55 times the number of shops you own in gold coins from the bank!"                     }
 };
 
 const QString &getVentureCardDesc(int ventureCard) {
-    return VENTURE_CARD_DESCS[ventureCard];
+    return VENTURE_CARD_TABLE[ventureCard].description;
+}
+
+bool isDefaultVentureCards(const bool ventureCards[], RuleSet ruleSet) {
```

Comment:
What do you mean by "default venture cards"?

Reply:
Its the venture card table that should be used if the imported yml/md file has no venture card table.
I defined the default by data mining the vanilla maps and choosing the venture cards which are used most frequently. Also no venture cards are enabled that depend on a specific tile being present (e.g. Boon, takeabreak, arcade)